### ISSUE TITLE
No text hints for save/save as/save all toolbar buttons

### ISSUE THREAD

**Issue by yshkp** [synfig/synfig#736]
While going through the software, I found out that beside the disability of Save All button preview of buttons are also not available of other saving buttons.
![screenshot_3](https://user-images.githubusercontent.com/39546036/53680888-4ef11a80-3d07-11e9-8c38-86f344f8e933.png)

**Comment by ankit-kumar-dwivedi** [comment]
> While going through the software, I found out that beside the disability of Save All button preview of buttons are also not available of other saving button

 @yshkp which version of Synfig are you using? 
Also your message is a bit unclear can you edit to elaborate more
Thanks :)

**Comment by morevnaproject** [comment]
My guess: Do you mean a text hints? (the ones, which appear when you hover on button)

**Comment by ankit-kumar-dwivedi** [comment]
@morevnaproject is the save all button disabled temporarily by Synfig? 

> My guess: Do you mean a text hints? (the ones, which appear when you hover on button)

Yes I just saw this :smile:

**Comment by morevnaproject** [comment]
@ankit-kumar-dwivedi This should be fixed in https://github.com/synfig/synfig/commit/a19e5589b6710e8d8e97eeb4c7686c600bcca06f

**Comment by yshkp** [comment]
@morevnaproject Yes, that text hints.

**Comment by yshkp** [comment]
@ankit-kumar-dwivedi Save All button is been disabled for the first tab.

**Comment by ankit-kumar-dwivedi** [comment]
@yshkp you many want to go ahead an add the description(text hints)! 
This is a easy fix.

**Comment by rockyhandsome1** [comment]
@morevnaproject  can I take this issue?

**Comment by yshkp** [comment]
@rockyhandsome1 Sorry, already started working on it.

**Comment by rockyhandsome1** [comment]
Oh I am really sorry, I waited for 10 minutes and thought of working on the issue, I created pull request.

**Comment by morevnaproject** [comment]
Hi everyone, apologies for delay.

@rockyhandsome1! Thank you for pull request. I can see it includes one redundant commit, which is not related to issue - 

![screenshot_001](https://user-images.githubusercontent.com/332868/53684803-ac578c80-3d44-11e9-91b3-9ed4857ec719.png)

Can you please clean it up?

**Comment by rockyhandsome1** [comment]
There are no pull requests in the forked repository that I have and cannot revert the same. As the initial commit (updated urls included)  is already merged.

**Comment by rockyhandsome1** [comment]
I have closed the pull request and will be generating a new pull request soon.

**Comment by morevnaproject** [comment]
> There are no pull requests in the forked repository that I have and cannot revert the same. As the initial commit (updated urls included) is already merged.

Note for future: you can update the branch (the one that you used as source for pull request) and relevant pull request will be updated automatically. ^__^

**Comment by morevnaproject** [comment]
@ankit-kumar-dwivedi @yshkp For future: if someone submits an issue and would like to work on it, then it would be nice to mention something like "I would like to work on  it." in issue description. I suggest to add this to Contributing Guidelines - https://synfig-docs-dev.readthedocs.io/en/latest/community/contribution%20guidelines.html

**Comment by morevnaproject** [comment]
@ankit-kumar-dwivedi @yshkp For future: if someone submits an issue and would like to work on it, then it would be nice to mention something like "I would like to work on  it." in issue description. I suggest to add this to Contributing Guidelines - https://synfig-docs-dev.readthedocs.io/en/latest/community/contribution%20guidelines.html

I hope this will help to avoid situations of double work too and gives issue author a priority to get the issue. ^__^

**Comment by morevnaproject** [comment]
@rockyhandsome1 The PR looks good now. Thank you! ^__^

**Comment by yshkp** [comment]
@morevnaproject I have made changes in Contributing Guidelines , please have a look.

**Comment by ankit-kumar-dwivedi** [comment]
> @ankit-kumar-dwivedi @yshkp For future: if someone submits an issue and would like to work on it, then it would be nice to mention something like "I would like to work on it." in issue description. I suggest to add this to Contributing Guidelines - https://synfig-docs-dev.readthedocs.io/en/latest/community/contribution%20guidelines.html
> 
> I hope this will help to avoid situations of double work too and gives issue author a priority to get the issue. ^__^

Changes already made sir :smiley:

**Comment by ankit-kumar-dwivedi** [comment]
> Oh I am really sorry, I waited for 10 minutes and thought of working on the issue, I created pull request.

Please do start working on issue someone is already working. We don't want to waste someone's work

Also @rockyhandsome1 please mention the issue number in the related PR so the issue closes automatically
see ->  https://help.github.com/en/articles/closing-issues-using-keywords

**Comment by ankit-kumar-dwivedi** [comment]
@morevnaproject please close this issue :)
one more issue solved :100:

**Comment by morevnaproject** [comment]
@ankit-kumar-dwivedi @yshkp Thank you both for the changes!

> if someone submits an issue and would like to work on it, then it would be nice to mention something like "I would like to work on it." in issue description. I suggest to add this to Contributing Guidelines

I suggest to add this to Contributing Guidelines. Mention that people can help by creating new issues. And while creating an issue author can let others know that he is willing to work on it. This is also a very good practice for devs - to create an issue before starting working on some task. ^__^

**Comment by ankit-kumar-dwivedi** [comment]
> @ankit-kumar-dwivedi @yshkp Thank you both for the changes!
> 
> > if someone submits an issue and would like to work on it, then it would be nice to mention something like "I would like to work on it." in issue description. I suggest to add this to Contributing Guidelines
> 
> I suggest to add this to Contributing Guidelines. Mention that people can help by creating new issues. And while creating an issue author can let others know that he is willing to work on it. This is also a very good practice for devs - to create an issue before starting working on some task. ^__^

yes, it's a good idea!
I am thinking to add https://github.com/synfig/synfig-tests-regressions/issues/3 from synfig-tests-regressions (as a starting point) it can be a beginner issue and every newcomer can add one file in this way he/she can get familiar with synfig.

**Comment by morevnaproject** [comment]
@ankit-kumar-dwivedi Thank you for making changes! Good point about https://github.com/synfig/synfig-tests-regressions/issues/3! ^__^
